web/src/components/Toast.tsx

```diff
@@ -1,82 +1,133 @@
 import * as ToastPrimitive from '@radix-ui/react-toast';
-import { X } from 'lucide-react';
+import { CircleCheck, Info, OctagonX, TriangleAlert, X } from 'lucide-react';
 import { useState } from 'react';
 import { useTranslation } from 'react-i18next';
+import { twMerge } from 'tailwind-merge';
 
 import { Button } from './Button';
 
-type Props = {
-  title: string;
-  description?: string;
+export enum ToastType {
+  INFO = 'info',
+  SUCCESS = 'success',
+  WARNING = 'warning',
+  DANGER = 'danger',
+}
```

Comment:
A thought just came to me, do we need to assign these string values or would pure number enums work?
```suggestion
export enum ToastType {
  INFO,
  SUCCESS,
  WARNING,
  DANGER,
}
```
I have no idea if this is better or not so curious to hear your thoughts about it.

Reply:
Either would work, but I went with strings in this case because I think it's more readable/semantic

Reply:
Makes sense! Let's keep them!